metpy/calc/kinematics.py

```diff
@@ -620,6 +624,42 @@ def lat_lon_grid_spacing(longitude, latitude, **kwargs):
     Accepts, 1D or 2D arrays for latitude and longitude
     Assumes [Y, X] for 2D arrays
 
+    .. deprecated:: 0.8.0
+    Function has been replaced with the signed delta distance calculation `lat_lon_grid_deltas`
```

Comment:
This needs to be indented (4 spaces) to make Sphinx happy--that's some of what Travis is unhappy about.